cli/flox/doc/flox-config.md

```diff
@@ -7,69 +7,112 @@ header: "flox User Manuals"
 
 # NAME
 
-flox-config - configure user parameters
+flox-config - view and set configuration options
 
 # SYNOPSIS
 
-flox [ `<general-options>` ] config ([(-l|\--list)] | (-r|\--reset) | --set `<key>` `<value>` | --set-number `<key>` `<number>` | --set-bool `<key>` `<bool>` | --delete `<key>`)
+```
+flox [ <general-options> ] config
+     [-l |
+      -r |
+      --set <key> <string> |
+      --set-number <key> <number> |
+      --set-bool <key> <bool> |
+      --delete=<key>]
+```
 
 # DESCRIPTION
 
-Configure and/or display user-specific parameters.
+Without any flags or when `-l` is passed, `flox config` shows all options with
+their computed value.
+
+The values are computed by reading:
+
+1. `flox` provided defaults.
+2. System settings from `/etc/flox.toml`.
+3. User customizations from `$XDG_CONFIG_DIRS`.
+4. User customizations from `$FLOX_CONFIG_HOME` if set or else `$XDG_CONFIG_HOME`.
+4. Environment variables.
+   All config options may be set by prefixing with `FLOX_` and using
+   SCREAMING_SNAKE_CASE.
+   For example, `disable_metrics` may be set with `FLOX_DISABLE_METRICS=true`.
 
-Without any flags or `-l` or `--list` shows all configurable options
-with their computed value.
+The last occurence is used as the final value.
+
+`flox config` commands that mutate configuration always write to
+`${FLOX_CONFIG_HOME:-$XDG_CONFIG_HOME}/flox/flox.toml`.
 
 ## Key Format
 
-`<key>` supports dot-separated queries for nested vaules, e.g.
+`<key>` supports dot-separated queries for nested values, for example:
 
 ```
-flox config --set 'nix.access_tokens."github.com"' "ghp_xxx"`
+flox config --set 'trusted_environments."owner/name"' trust
 ```
 
-## Config Format
+# OPTIONS
 
-All config keys can be listed with
+## Config Options
 
-```
-flox config
-```
+`-l`, `--list`
+:   List the current values of all options.
 
-The dispalyed values are resolved by reading:
+`-r`, `--reset`
+:   Reset all options to their default values without further confirmation.
 
-1. package defaults from `$PREFIX/etc/flox.toml`
-2. installation defaults from `/etc/flox.toml`
-3. user customizations from `$HOME/.config/flox/flox.toml`
-4. environment variables
+`--set <key> <value>`
+:  Set `<key> = <value>` for string values
 
-where the last occurence is used as the final value.
+`--set-number <key> <value>`
+:  Set `<key> = <value>` for number values
 
-`flox config` commands that mutate configurations always write to `$HOME/.config/flox/flox.toml`.
+`--set-bool <key> <value>`
+:  Set `<key> = <value>` for boolean values
 
-# OPTIONS
+`--delete <key>`
+:   Delete config key
 
 ```{.include}
 ./include/general-options.md
-./include/development-options.md
 ```
 
-## Config Options
+# SUPPORTED CONFIGURATION OPTIONS
```

Comment:
I personally think it's simpler to keep it all here since I'm expecting we document usage of `flox config` more than we document editing `flox.toml`. But we could split it out 🤷

Reply:
I think we should just keep it here